assets/src/main/java/bisq/asset/ErgoAddressValidator.java

```diff
@@ -0,0 +1,58 @@
+/*
+ * This file is part of Bisq.
+ *
+ * Bisq is free software: you can redistribute it and/or modify it
+ * under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or (at
+ * your option) any later version.
+ *
+ * Bisq is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero General Public
+ * License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with Bisq. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package bisq.asset;
+
+import java.util.Arrays;
+
+import org.bitcoinj.core.Base58;
+import org.bitcoinj.core.AddressFormatException;
+import org.bouncycastle.crypto.digests.Blake2bDigest;
+
+/**
+ * {@link AddressValidator} for Base58-encoded Ergo addresses.
+ */
+public class ErgoAddressValidator implements AddressValidator {
```

Comment:
Please make this validator an inner class of Ergo class.

Reply:
Done!